docs/usage-as-end-user.md

```diff
@@ -61,6 +61,14 @@ apiVersion: packet.provider.extensions.gardener.cloud/v1alpha1
 kind: ControlPlaneConfig
 ```
 
+An example `ControlPlaneConfig` with persistence neabled, using rook-ceph, looks as follows:
```

Comment:
typo

Also, maybe you can add more information about when to enable/disable it (what's the default?).

Reply:
Sure. Done.